generators/server/generator.mjs

```diff
@@ -40,126 +41,149 @@ export default class extends GeneratorBaseEntities {
       },
 
       async pomXml({ application: { buildToolMaven } }) {
```

Comment:
It would be better to extract gradle to another task instead of reusing pomXml.
IMO keeps a bit easier to understand.
```suggestion
      async customizeGradle({ application: { buildToolGrade } }) {
        if (! buildToolGrade) return;
      }

      async customizeMaven({ application: { buildToolMaven } }) {
```